Hazel/Dtls/DtlsConnectionListener.cs

```diff
@@ -1283,6 +1287,29 @@ protected override void QueueRawData(ByteSpan span, IPEndPoint remoteEndPoint)
             }
         }
 
+        /// <inheritdoc />
+        public override void DisconnectOldConnections(TimeSpan maxAge, MessageWriter disconnectMessage)
```

Comment:
This method is actually a weird vestige from failed anti-abuse, I don't actually use it and I'd be surprised if anyone really does. That lends me to think that it should be private to this class, maintained ala reliablePacketThread, remove the base implementation and make NegotiationTimeout a public property with a sensible default.

Reply:
Actually. Apparently I do use this, so I retract my statement entirely.